src/components/TeamViewer.js

```diff
@@ -0,0 +1,22 @@
+import React, {Component} from 'react';
+
+export default class TeamViewer extends Component{
+    constructor(props){
+        super(props);
+    }
```

Comment:
Same comment as above. This doesn't need the constructor, and could probably be made into a stateless function.